src/ORM/EntityManager.php

```diff
@@ -46,6 +46,43 @@ public function flush(): void
         $this->callMethod('flush');
     }
 
+    /**
+     * Generate a unique value based on provided field.
+     *
+     * @param \EoneoPay\Externals\ORM\Interfaces\RepositoryInterface $repository
+     * @param string $field
+     * @param int|null $length
+     *
+     * @return string
+     *
+     * @throws \EoneoPay\Externals\ORM\Exceptions\ORMException
+     */
+    public function generateRandomUniqueValue(
+        RepositoryInterface $repository,
```

Comment:
The only reason I changed it from Entity to repository is because the only usage of the Entity in this function was to obtain the repository from it, but I can see what you mean because the context is more appropriate to be the entity rather than `Repository`.
I will fix this up.